ht/cogs/roll.py

```diff
@@ -45,6 +49,60 @@ async def armiger(self, ctx, user: converters.Armiger = None):
 		await self.add_rolls(embed, "AND NOT personal", user, "Artist gallery")
 		await ctx.send(embed = embed)
 
+	@commands.command(
+		help = "Looks up the symbology of a user's coat of arms.\nUses GreiiEquites' https://roll-of-arms.com as a source.",
+		aliases = ("s", "symbols")
+	)
+	async def symbolism(self, ctx, user: converters.Armiger = None):
+		if not user:
+			user = await ctx.bot.dbc.execute_fetchone(
+				"SELECT * FROM armigers_e WHERE discord_id == ?;", (ctx.author.id,)
+			)
+
+			if not user:
+				await ctx.typing()
+				await self.bot.get_cog("Bot tasks").sync_book()
+
+				user = await ctx.bot.dbc.execute_fetchone(
+					"SELECT * FROM armigers_e WHERE discord_id == ?;", (ctx.author.id,)
+				)
+
+			if not user: raise utils.CustomCommandError(
+				"Invalid armiger",
+				"There are no arms associated with your user account. "
+				"To find those of another user, follow the command with their username."
+				"If you wish to register your arms, follow the instructions at the Roll of Arms server."
+			)
+
+			async with aiohttp.ClientSession() as session:
+				async with session.get(f'https://roll-of-arms.com/wiki/GreiiN:{user[0]}') as response:
+					if response.status == 404:
+						utils.CustomCommandError(
+							"Armiger is not on roll-of-arms.com",
+							"The arms of the armiger are not on the https://roll-of-arms.com "
+							"website. If you would like to add your arms and related symbolism "
+							"to the website, please fill out the form pinned in the "
+                            "#announcements channel of the Roll of Arms server."
+						)
+
+					soup = BeautifulSoup(await response.text(), 'html.parser')
+					value = soup.select('h2:has(#Symbolism)')[0]
+					next_section = value.next_sibling
+					symbolism_text = ''
+					while next_section is not None and not isinstance(next_section, Comment) and not str(next_section).startswith('<h'):
+						markdown = re.sub(
+							r'<[^>]*>',
+							'',
+							next_section.replace('<b>', '**').replace('</b>', '**').replace('<i>', '*').replace('</i>', '*')
+						)
+						symbolism_text += f"{markdown}\n"
+						next_section = next_section.next_sibling
+
+					embed = embeds.GENERIC.create(f'{user[2]}#{user[3]:04}', symbolism_text, heading = f'GreiiN:{user[0]:04}')
+					embed.set_footer(text = 'Textual content from https://roll-of-arms.com by GreiiEquites.')
+
+					await ctx.send(embed = embed)
+
```

Comment:
```suggestion
				
			embed = embeds.GENERIC.create(f"Symbolism of {user[2]}#{user[3]:04}", symbolism_text, heading = f'GreiiN:{user[0]:04}')
			embed.set_footer(text = "Textual content from https://roll-of-arms.com by GreiiEquites.")
			
			await ctx.send(embed = embed)

```

Add "Symbolism" to the heading to make it more distinct from using `!a`